### PR TITLE
fix ESG link behavior from instructor dashboard when ESG flag enable

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** Fix bug for openassessment title is leading to ESG link instead of ORA.

JIRA: [AU-576](https://openedx.atlassian.net/browse/AU-576)

**What changed?**

- Revert the change of the older URL logic
- Create ESGCell logic and stop reusing older logic that is overlapping with other column as well.
- Update unit test
- **Minor** `load_static` to help with hot reload. I didn't intend it to be here but accidentally commit it already.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

make sure [ENABLE_XBLOCK_VIEW_ENDPOINT](https://github.com/openedx/edx-platform/blob/29e50710dc2b59ad73fa5234e32b07242a6f0411/lms/envs/common.py#L227) is set to true.

- Disabled ESG
  - Go to [http://localhost:18000/admin/waffle/flag/](http://localhost:18000/admin/waffle/flag/)
  - add/update Flag
    - name `openresponseassessment.enhanced_staff_grader`
    - change everyone to `no`
    - check `staff`
    - Save
  - [ ] Go to `instructor dashboard -> Open Responses`
  - [ ] There shouldn't be additional column
  - [ ] Staff column link should lead to in-window legacy staff grader
  - [ ] Assessment column link should lead to in-window ora 
- Enabled ESG
  - Go to [http://localhost:18000/admin/waffle/flag/](http://localhost:18000/admin/waffle/flag/)
  - add/update Flag
    - name `openresponseassessment.enhanced_staff_grader`
    - change everyone to `yes`
    - check `staff`
    - Save
  - [ ] Go to `instructor dashboard -> Open Responses`
  - [ ] There should be additional column title response with link to new page of ESG
  - [ ] Staff column link should lead to new page of ESG
  - [ ] **Previous bug**: Assessment column link should remain link to in-window ora

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/xblock/static/js/src/lms/oa_course_items_listing.js [hunk 1]
```diff
@@ -144,12 +129,43 @@ export class CourseItemsListingView {
       },
     ];
 
+    const esgEnabled = context.ENHANCED_STAFF_GRADER;
+    const esgRootUrl = context.ORA_GRADING_MICROFRONTEND_URL;
+
     if (itemViewEnabled && esgEnabled) {
```
Comment (leangseu-edx) [inline openassessment/xblock/static/js/src/lms/oa_course_items_listing.js:135]: I am just trying to move all the logic into a single place.

**Review by nsprenkle — CHANGES_REQUESTED** [review]
The links are no longer working for me when I test locally:

Screen Shot 2022-03-28 at 14 51 14

**Review by nsprenkle — APPROVED** [review]
